packages/pwa-kit-dev/src/utils/script-utils.ts

```diff
@@ -255,9 +283,26 @@ export const createBundle = async ({
                     })
             )
             .then(async () => {
-                const {dependencies = {}, devDependencies = {}} = await getProjectPkg()
+                const {
+                    dependencies = {},
+                    devDependencies = {},
+                    ccExtensibility = {extends: '', overridesDir: ''}
+                } = await getProjectPkg()
+                const extendsTemplate = 'node_modules/' + ccExtensibility.extends
```

Comment:
```suggestion
                const extendsTemplate = require.resolve(ccExtensibility.extends)
```
We shouldn't assume that the package lives in `$PWD/node_modules` - there's some weird edge cases that make `require.resolve` a bit more reliable. (I don't know how likely we are to hit those edge cases, but why worry about it? 🤷)